.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+Thank you for contributing to Harbor!
+
+# Comprehensive Summary of your change
+
+# Issue being fixed
+Fixes #(issue)
+
+Please indicate you've done the following:
+- [ ] Well Written Title and Summary of the PR
+- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
+- [ ] Accepted the DCO. Commits without the DCO will delay acceptance.
```

Comment:
Be honest here: PRs without a DCO are not accepted. At least I never saw one.

Reply:
we have CI to guarantee it.